go-controller/pkg/ovn/external_ids_syncer/acl/acl_sync_test.go

```diff
@@ -749,4 +749,91 @@ var _ = ginkgo.Describe("OVN ACL Syncer", func() {
 		}
 		testSyncerWithData(testData, controllerName, []libovsdbtest.TestData{}, nil, nil)
 	})
+	ginkgo.It("deletes leftover multicast acls", func() {
```

Comment:
for the test I wish there were more comments explaining exactly what you are creating and the inline comments for the steps on what you are testing. Its hard to follow, and I dont get why the tier is updated, but I trust you did the test case correctly so will merge it.